dist/index.js

```diff
@@ -73173,8 +73173,12 @@ async function addComment(client, taskId, comment) {
     (0,_actions_core__WEBPACK_IMPORTED_MODULE_0__.info)(`Added the GitHub link to the Asana task: ${taskId}`);
 }
 function getPreviousText() {
+    var _a, _b;
```

Comment:
For a very brief second I was like "Hiranya... why?" then I realized what this file actually was.